### ISSUE TITLE
MA pan balance

### ISSUE THREAD

**Issue by sum1won** [sum1won/summod#125]
Pan cost increase, ?? STR black harpies?

**Comment by KimmoKM** [comment]
It's worth noting llwinter making revelers undisciplined was a pretty big nerf already yet to be reflected in win rates (not as huge for MA Pan as it is for EA seeing as that your high number of E3 Panii can get buffs up turn 1, but a far from inconsequential change).

I like STR black harpies, though. I'd rather not increase the cost of Panii seeing as that they're already expensive (it's just that they have great synergy with revelers), if mage costs were to be increased I'd rather target dryads. They don't have much use other than being seducer assassins (and some people prefer to use them in expansion) and besides being really annoying to play against, thematically I don't think that's really "the thing" for Pangaea anyway but in practise it's a big part of the nations strength.

**Comment by Lambert-Simnel** [comment]
More unrest? Dominion based unrest? Ends up making blood less easy for them, as well as reducing income, fairly thematically.

**Comment by KimmoKM** [comment]
I disagree about that being thematic. In my view, MA Pangaea is at crossroads where they can fully embrace civilization (the "canon" choice) or stubbornly cling to the old ways of revelry. If you do the former, unrest would definitely be unthematic, and in the latter case it's already represented by unrest being generated by satyrs. If you were to use that sort of idea, Pandemoniacs could generate unrest (and that I could get behind, seeing as that a lot of Pangaean population already abhors blood magic, with Panii having stopped using it and all), but unrest dominion? I don't like it at all.

However, that gave me an idea to another possibility: EA Pan is "The Age of Revelry" but MA is in fact far superior at using revelers due to their better access to earth buffs. What about if there was a slight nerf to MA revelers to represent their dying ways (like -1 or -2 berserker)?

**Comment by master-bizon** [comment]
As a recent MA Pangaea winner I'd like to chime in. (Lambert was in it - MemeOfTheAges)
Pangaea has a very strong earlygame and midgame, however drops off in the lategame - I was last in Research by the time I won.

In the early game you lean heavily on your blessed centaurs, probably the best units in the game at this point, to win an early war.

In the midgame, you want to use Destruction/swarm/wooden warriors/AoE 9 quickness(if it's still in summod) as battlemagic. With swarm online, each dryad is capable of utterly ruining enemy scripts by killing off critical commanders. Dryads are capable of winning wars almost on their own.

In the lategame the fact that you have ineffective research catches up to you and you need to lean heavily on dryads to keep any enemy advances off of you.

The reveler undisciplined nerf has made them difficult to buff. It also ruined one of my favorite raiding configurations (Protection)x4 on 10-15 revelers.

As for the suggested nerfs so far:
**Pan cost increase** - Pangaea already has one of the more expensive mages around. I don't think Pans are the biggest problem.
**STR Harpies** - Map dependent if it's impactful or not, the good scouting of Pan is strong, but I don't think it's what makes the nation oppressive.
**Dominion unrest** - as Kimmo said I don't think it's thematic +blood is largely irrelevant, just like in EA Pan. And giving a nation unrest "because it's strong now" seems like bad design.

I think the main difficulty of playing against Pangaea is dealing with early W9 centaurs and fighting into dryads. However if you survive into the lategame you should have enough tools to deal with what Pangaea has.

If you wanted to hit Pangaea hard, I'd go for the dryads. It allows Pangaea to win wars effectively outside of the battlefield.  They're hard to deal with, a N2 dryad can spam swarm so much it kills a mage with a living water bottle equipped. Make dryads StR? Or perhaps a more proactive solution, create an item which allows a commander to survive swarm spam? (Does charcoal shield do this effectively?) Or make such items more accessible akin to how supply items were lowered to level 0, items to survive swarmspam assasinations could be made more accessible?

The reasons I like playing Pangaea are:
- Good scouting (Flying, MM3, recruited from **any forest**)
- Solid troops(Revelers) worth recruiting and buffing with my Nature magic, a synergy with my Nature mages.
- Dryads providing a third vector of attack in the form of seductions. 

This is what makes Pangaea the nation it is for me.

If you made the dryads unusable I'm pretty sure I would not pick this nation up anytime soon. But I think they are very hard to counter when used effectively.

And lastly, does Pangaea need a nerf? It's really strong early-mid but falls off later on. Is the nation that oppressive in all the games it is in?
